.github/checkgroup.yml

```
@@ -26,74 +26,74 @@ subprojects:
       - ".actions/**"
     checks:
       # Note: updates here should be applied to the lightning_lite group
-      - "pl-cpu (macOS-11, pytorch, 3.8, 1.10)"
-      - "pl-cpu (macOS-11, pytorch, 3.9, 1.11)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.12)"
-      - "pl-cpu (macOS-11, pytorch, 3.7, 1.9, oldest)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.13, pre)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.8, 1.10)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.9, 1.11)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.10, 1.11)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.10, 1.12)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.7, 1.9, oldest)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.9, 1.13, pre)"
-      - "pl-cpu (windows-2022, pytorch, 3.9, 1.11)"
-      - "pl-cpu (windows-2022, pytorch, 3.10, 1.11)"
-      - "pl-cpu (windows-2022, pytorch, 3.10, 1.12)"
-      - "pl-cpu (windows-2022, pytorch, 3.7, 1.9, oldest)"
-      - "pl-cpu (windows-2022, pytorch, 3.8, 1.13, pre)"
-      - "pl-cpu (macOS-11, lightning, 3.10, 1.12)"
+      - "pl-cpu (quick, macOS-11, pytorch, 3.8, 1.10)"
```

The old jobs should be unchanged

```suggestion
      - "pl-cpu (macOS-11, pytorch, 3.8, 1.10)"
```

---

resolved in https://github.com/Lightning-AI/lightning/pull/15331#discussion_r1011730910